tests/www/employee_record_views/test_list.py

```diff
@@ -175,7 +175,7 @@ def test_employee_records_with_nir_associated_to_other(self, client, snapshot):
             parse_response_to_soup(
                 response,
                 selector=".employee-records-list .c-box--results__footer",
-                replace_in_attr=[self.employee_record, self.job_application],
+                replace_in_attr=[self.job_application, self.employee_record],
```

Comment:
Je crois que c'est effectivement un truc du style que j'avais fait, mais là je me dit que c'est assez fragile donc peut-être plutôt :
```suggestion
                replace_in_attr=[
                    (
                        "href",
                        f"https://tally.so/r/wzxQlg?employeerecord={self.employee_record.pk}&jobapplication={self.job_application.pk}",
                        "https://tally.so/r/wzxQlg?employeerecord=[PK of EmployeeRecord]&jobapplication=[PK of JobApplication]",
                    )
                ],
```
Ou
```suggestion
                replace_in_attr=[
                    (
                        "href",
                        f"https://tally.so/r/wzxQlg?employeerecord={self.employee_record.pk}&jobapplication={self.job_application.pk}",
                        "[TALLY LINK]",
                    )
                ],
      ```

Reply:
:+1: 
 `parse_response_to_soup` est plus flexible que ce que je pensais !